controller/api/destination/server.go

```diff
@@ -7,6 +7,8 @@ import (
 	"strconv"
 	"strings"
 
+	coreinformers "k8s.io/client-go/informers/core/v1"
+
```

Comment:
nit: Can you remove the newline below this so it gets group with the other imports?

Reply:
Sure thing, removed the new line and resorted the imports to make the lint pass.